scripts/helper.sh

```diff
@@ -38,3 +38,67 @@ function fail_if_error() {
     exit $retVal
   fi
 }
+
+function check_image_artifact_exists() {
+  local FILE=$1
+  if [[ ! -f "$FILE" ]]; then
+    echo "Error: Image artifact $FILE does not exist."
```

Comment:
Strictly speaking, failing the -f check could mean it exists but is a directory or a device or something.
```suggestion
    echo "Error: Image artifact $FILE does not exist or is not a regular file."
```